stripe/src/test/java/com/stripe/android/StripeKtxTest.kt

```diff
@@ -0,0 +1,552 @@
+package com.stripe.android
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import com.stripe.android.exception.AuthenticationException
+import com.stripe.android.exception.InvalidRequestException
+import com.stripe.android.model.PaymentIntent
+import com.stripe.android.model.SetupIntent
+import com.stripe.android.model.Source
+import com.stripe.android.networking.ApiRequest
+import com.stripe.android.networking.StripeApiRepository
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import kotlin.test.assertFailsWith
+import kotlin.test.assertSame
+
+/**
+ * Test for [Stripe] suspend functions.
+ */
+@ExperimentalCoroutinesApi
+@RunWith(JUnit4::class)
+internal class StripeKtxTest {
+    private val mockApiRepository: StripeApiRepository = mock()
+    private val mockPaymentController: PaymentController = mock()
+
+    private val stripe: Stripe =
```

Comment:
You'll need to cleanup the `TestCoroutineDispatcher` after the test run, so you'll need to keep an instance of it around.

```
    @AfterTest
    fun cleanup() {
        testDispatcher.cleanupTestCoroutines()
    }
```